File: R Programming files/Bubble_Sort.R
my_array = c(4,7,2,5,9,12)

sorting_func = function(my_array) {
  n = length(my_array)
  
  for (i in 1:(n - 1)) {
    for (j in 1:(n - i)) {
      if (my_array[j] > my_array[j + 1]) {
        temporary = my_array[j]
        my_array[j] = my_array[j + 1]
        my_array[j + 1] = temporary
      }
    }
  }
  print(my_array)
}

sorting_func(my_array)

File: Bubble_Sort.py
my_array = [4,7,2,5,9,12]

def sorting_func():
    swap = True
    while (swap == True):
        swap = False
        for i in range(len(my_array) - 1):
            
            if (my_array[i] > my_array[i + 1]):
                swap_num = my_array[i]
                my_array[i] = my_array[i + 1]
                my_array[i + 1] = swap_num
                swap = True
    return my_array
        
sorted_array = sorting_func()           
print(sorted_array)